c/zis/build.sh

```diff
@@ -88,7 +88,7 @@ as "${ASFLAGS[@]}" -aegimrsx=nwm.asm nwm.s
 as "${ASFLAGS[@]}" -aegimrsx=secmgmt.asm secmgmt.s
 as "${ASFLAGS[@]}" -aegimrsx=snarfer.asm snarfer.s
 
-export _LD_SYSLIB="//'SYS1.CSSLIB'://'CEE.SCEELKEX'://'CEE.SCEELKED'://'CEE.SCEERUN'://'CEE.SCEERUN2'://'CSF.SCSFMOD0'"
+export _LD_SYSLIB="//'SYS1.CSSLIB'://'CEE.SCEELKEX'://'CEE.SCEELKED'://'CEE.SCEERUN'://'CEE.SCEERUN2'://'PP.CSF.ZOS203.SCSFMOD0'"
```

Comment:
This shouldn't have been changed.

Reply:
You're right; it was for the local build.  I will revert it.